src/atoms/Link/__tests__/Link.web.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import Link from '../index';
+import { renderWithTheme } from '../../../_helpers/testing';
+
+describe('<Link />', () => {
+  it('renders a link', () => {
+    const displayText = 'Displaying some link';
+    const { container } = renderWithTheme(<Link>{displayText}</Link>);
+    expect(container).toMatchSnapshot();
+  });
+  describe('Size', () => {
+    it('small', () => {
+      const displayText = 'Displaying some link';
+      const { container } = renderWithTheme(<Link size="small">{displayText}</Link>);
+      expect(container).toMatchSnapshot();
+    });
+
+    it('medium', () => {
+      const displayText = 'Displaying some link';
+      const { container } = renderWithTheme(<Link size="medium">{displayText}</Link>);
+      expect(container).toMatchSnapshot();
+    });
+
+    it('large', () => {
+      const displayText = 'Displaying some link';
+      const { container } = renderWithTheme(<Link size="large">{displayText}</Link>);
+      expect(container).toMatchSnapshot();
+    });
+  });
+  describe('Selectors', () => {
+    it('onHover', () => {
+      const displayText = 'Displaying some link';
```

Comment:
Can't see test cases for other selectors? active, visited, focus

Reply:
updated for active and focus. 
Have any idea how to do it for visited?